src/cli/ExampleProjectCommand.ts

```diff
@@ -28,24 +28,108 @@ export class ExampleProjectCommand extends DataModelerCliCommand {
 
     console.log("Downloading dataset for example project...");
     execSync(
-      `curl -s http://pkg.rilldata.com/rill-developer-example/data/flightlist.zip ` +
-        `--output ${project}/flightlist.zip`,
+      `curl -s http://pkg.rilldata.com/rill-developer-example/example-assets.zip ` +
+        `--output ${project}/example-assets.zip`,
       { stdio: "inherit" }
     );
-    execSync(`unzip ${project}/flightlist.zip ` + `-d ${project}/`, {
+    execSync(`unzip ${project}/example-assets.zip ` + `-d ${project}/`, {
       stdio: "inherit",
     });
 
-    console.log("Importing example dataset into the project...");
+    console.log("Importing example datasets into the project...");
+    console.log("Adtech...");
+    await new ImportTableCommand().run(
+      {
+        projectPath: project,
+        profileWithUpdate: true,
+      },
+      `${project}/example-assets/data/adtech-ad-click/adtech-item-data.csv`,
+      {}
+    );
+    await new ImportTableCommand().run(
+      {
+        projectPath: project,
+        profileWithUpdate: true,
+      },
+      `${project}/example-assets/data/adtech-ad-click/adtech-train.csv`,
+      {}
+    );
+    await new ImportTableCommand().run(
+      {
+        projectPath: project,
+        profileWithUpdate: true,
+      },
+      `${project}/example-assets/data/adtech-ad-click/adtech-view-log.csv`,
+      {}
+    );
+
+    console.log("Crypto...");
+    await new ImportTableCommand().run(
+      {
+        projectPath: project,
+        profileWithUpdate: true,
+      },
+      `${project}/example-assets/data/crypto-bitcoin/crypto-bitstamp-usd.csv`,
+      {}
+    );
+
+    console.log("Ecommerce...");
     await new ImportTableCommand().run(
       {
         projectPath: project,
         profileWithUpdate: true,
       },
-      `${project}/data/flightlist_2022_02.csv`,
+      `${project}/example-assets/data/ecomm-click-stream/e-shop-clothing.csv`,
       {}
     );
 
+    console.log("Global...");
+    await new ImportTableCommand().run(
+      {
+        projectPath: project,
+        profileWithUpdate: true,
+      },
+      `${project}/example-assets/data/global-landslide-catalog/global-landslide-catalog.csv`,
+      {}
+    );
+
+    console.log("Internet of Things...");
+    await new ImportTableCommand().run(
+      {
+        projectPath: project,
+        profileWithUpdate: true,
+      },
+      `${project}/example-assets/data/iot-env-sensor/iot-telemetry-data.csv`,
+      {}
+    );
+
+    console.log("Importing example SQL transformations into the project...");
+    execSync(`mv -v ${project}/example-assets/models/* ${project}/models`, {
```

Comment:
Replace these mv and rm with nodejs APIs ? That way it will work on windows more easily (we still need an alternative to unzip). @magorlick If not we should mention that our app doesnt work on windows just yet.

Reply:
This is a good call out. We will leave this as is for the next release and follow up when we tackle https://github.com/rilldata/rill-developer/issues/510.   I will make a user note in our docs that we are Mac and Linux only.